packages/itmat-ui-react/src/components/users/index.tsx

```diff
@@ -26,6 +27,14 @@ export const UserPage: React.FunctionComponent = (props) => {
                             </div>
                         </>
                     } />
+                    <Route path="/users/manageBatchUsers" render={() =>
+                        <>
+                            <div className="page_ariane">CREATE NEW USER</div>
```

Comment:
Why does this have the same title as the other category ?

Reply:
Sorry it is a mistake, would correct it.